sentry_streams/src/transformer.rs

```diff
@@ -100,4 +117,70 @@ mod tests {
             assert_messages_match(py, expected_messages, actual_messages.deref());
         });
     }
+
+    #[test]
```

Comment:
I'd put this in the filter_step module as rust tests go together with the code they test.